polyfills/HTMLInputElement/prototype/valueAsDate/config.toml

```
@@ -0,0 +1,39 @@
+#### Aliases
+aliases = []
+
+#### Dependencies
+dependencies = [
+	"Object.defineProperty"
```

On line 11 of the polyfill there is a called to Object.getOwnPropertyDescriptor - https://github.com/Financial-Times/polyfill-library/pull/1043/files#diff-dec27411bbff16e8942db416d17c042db40eabaa178872c3fd5a87767370db95R11

We should declare it as a dependency for browser versions like Firefox 35 and lower which do not have it natively
```suggestion
	"Object.defineProperty",
	"Object.getOwnPropertyDescriptor"
```